frontend/glideinFrontendMonitorAggregator.py

```diff
@@ -457,15 +382,15 @@ def aggregateStatus():
     # Write rrds
 
     glideinFrontendMonitoring.monitoringConfig.establish_dir("total")
-    write_one_rrd("total/Status_Attributes", updated, global_total, 0)
+    write_one_rrd(os.path.join("total", "Status_Attributes"), updated, global_total, 0)
 
     for fact in list(global_fact_totals["factories"].keys()):
-        fe_dir = "total/factory_%s" % glideinFrontendMonitoring.sanitize(fact)
+        fe_dir = os.path.join("total", "factory_" + glideinFrontendMonitoring.sanitize(fact))
```

Comment:
Will it be better to change to `f"factory_{glideinFrontendMonitoring.sanitize(fact)}"`?

Reply:
done